l10n_fr_ecotaxe/models/ecotaxe_line_mixin.py

```diff
@@ -0,0 +1,58 @@
+# © 2014-2023 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class EcotaxeLineMixin(models.AbstractModel):
+    """Mixin class for objects which can be used to save
+     multi ecotaxe calssification  by account move line
+    or sale order line."""
+
+    _name = "ecotaxe.line.mixin"
+    _description = "Ecotaxe Line Mixin"
+
+    product_id = fields.Many2one("product.product", string="Product", readonly=True)
+    currency_id = fields.Many2one("res.currency", string="Currency")
+    ecotaxe_classification_id = fields.Many2one(
```

Comment:
Because it's an ecotaxe model, maybe `classification_id` name is sufficient here